.github/workflows/auto-merge.yml

```
@@ -0,0 +1,31 @@
+name: auto-merge
+
+on:
+  pull_request:
+    branches: [ main ]
+    types: [ labeled ]
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  auto-merge:
+    runs-on: ubuntu-latest
+    if: >-
+      github.event.pull_request.user.login == 'Hiroshiba' &&
+      github.event.pull_request.labels[0].name == 'bump-cask-pr'
```

0番目以外にも反応するように、ついているラベルの中に指定したものがあるかどうか調べる書き方があるっぽいです！
https://stackoverflow.com/a/71510189
```suggestion
      contains(github.event.pull_request.labels.*.name, 'bump-cask-pr')
```

---

これ知りませんでした
ありがとうございます！